ci/terraform/oidc/variables.tf

```diff
@@ -611,6 +611,12 @@ variable "kms_cross_account_access_enabled" {
   description = "Whether the service should allow cross-account access by the orchestration account to kms"
 }
 
+variable "txma_audit_encoded_enabled" {
```

Comment:
Could you create a followup ticket to remove this once we're live everywhere please?